squashfs/index.html

```diff
@@ -1117,13 +1117,13 @@ <h4 id="directory-entry">Directory Entry</h4>
                                 <td>The <a href="#inode-types">inode type</a>. <b>For extended inodes, the corresponding basic type is stored here instead<b></td>
                             </tr>
                             <tr>
-                                <td>size</td>
+                                <td>name_size</td>
                                 <td>u16</td>
                                 <td>The size of the entry name</td>
```

Comment:
I think this should be something like `one less than the size of the file name in bytes` (same for below)